carbon_tracker_workflow.ipynb

```diff
@@ -0,0 +1,2510 @@
+{
```

Comment:
<p>The values aren't converted anymore</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EXCITED-CO2/excited-workflow/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>